src/Uno.UI/Microsoft/UI/Xaml/Controls/NavigationView/NavigationViewTemplateSettings.Properties.cs

```diff
@@ -1,173 +1,172 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
-// MUX Reference NavigationViewTemplateSettings.properties.cpp, commit 991c831
+// MUX Reference NavigationViewTemplateSettings.properties.cpp, commit bace2c5
 
 using Windows.UI.Xaml;
 
-namespace Microsoft.UI.Xaml.Controls
+namespace Microsoft.UI.Xaml.Controls;
+
+/// <summary>
+/// Provides calculated values that can be referenced as TemplatedParent sources
+/// when defining templates for a NavigationView. Not intended for general use.
+/// </summary>
+public partial class NavigationViewTemplateSettings : DependencyObject
 {
 	/// <summary>
-	/// Provides calculated values that can be referenced as TemplatedParent sources
-	/// when defining templates for a NavigationView. Not intended for general use.
+	/// Initializes a new instance of the NavigationViewTemplateSettings class.
+	/// </summary>
+	public NavigationViewTemplateSettings()
+	{
+	}
+
+	/// <summary>
+	/// Gets the visibility of the back button.
+	/// </summary>
+	public Visibility BackButtonVisibility
+	{
+		get => (Visibility)GetValue(BackButtonVisibilityProperty);
+		internal set => SetValue(BackButtonVisibilityProperty, value);
+	}
+
+	/// <summary>
+	/// Identifies the BackButtonVisibility dependency property.
+	/// </summary>
+	public static DependencyProperty BackButtonVisibilityProperty { get; } =
+		DependencyProperty.Register(nameof(BackButtonVisibility), typeof(Visibility), typeof(NavigationViewTemplateSettings), new FrameworkPropertyMetadata(Visibility.Collapsed));
+
+	/// <summary>
+	/// Gets the visibility of the left pane.
+	/// </summary>
+	public Visibility LeftPaneVisibility
+	{
+		get => (Visibility)GetValue(LeftPaneVisibilityProperty);
+		internal set => SetValue(LeftPaneVisibilityProperty, value);
+	}
+
+	/// <summary>
+	/// Identifies the LeftPaneVisibility dependency property.
+	/// </summary>
+	public static DependencyProperty LeftPaneVisibilityProperty { get; } =
+		DependencyProperty.Register(nameof(LeftPaneVisibility), typeof(Visibility), typeof(NavigationViewTemplateSettings), new FrameworkPropertyMetadata(Visibility.Visible));
+
+	/// <summary>
+	/// Gets the width of open pane.
+	/// </summary>
+	public double OpenPaneLength
+	{
+		get => (double)GetValue(OpenPaneLengthProperty);
+		set => SetValue(OpenPaneLengthProperty, value);
```

Comment:
Won't this be a breaking change?

Reply:
The removal of `OpenPaneWidth` is a breaking change, but the `OpenPaneLength` property was there already generated by Sync Gen. WinUI did the rename when the API became stable.